### PR TITLE
databases-week1

### PR DESCRIPTION
**Pull request by Rajesh-Bhatt**
Queries designed, executed and result verified for the following:

1. Find out how many tasks are in the task table
2. Find out how many tasks in the task table do not have a valid due date
3. Find all the tasks that are marked as done
4. Find all the tasks that are not marked as done
5. Get all the tasks, sorted with the most recently created first
6. Get the single most recently created task
7. Get the title and due date of all tasks where the title or description contains database
8. Get the title and status (as text) of all tasks
9. Get the name of each status, along with a count of how many tasks have that status
10. Get the names of all statuses, sorted by the status with most tasks first.

### CODE REVIEW

File: databases/week1/queries.sql [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/*Find out how many tasks are in the task table*/
+SELECT COUNT(*) FROM task;
+/*Find out how many tasks in the task table do not have a valid due date*/
+SELECT * FROM task WHERE due_date IS NULL;
```
Comment (jpganz) [inline databases/week1/queries.sql]: this query is wrong, please take a look

File: databases/week1/queries.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+/*Find all the tasks that are marked as done*/
+SELECT * FROM task WHERE status_id = 3;
```
Comment (jpganz) [inline databases/week1/queries.sql:10]: you are assuming Done has id 3, but you can also get it by the title

**Review by jpganz — CHANGES_REQUESTED** [review]

File: databases/week1/queries.sql (hunk 1 above)
Comment (Rajesh-Bhatt) [reply databases/week1/queries.sql]: Hi jpganz, 
This query has been improved now. Thank you for your review!

File: databases/week1/queries.sql (hunk 2 above)
Comment (Rajesh-Bhatt) [reply databases/week1/queries.sql:10]: Yes, we can do that also. Thank you for your suggestion.

File: databases/week1/queries.sql [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/*Find out how many tasks are in the task table*/
+SELECT COUNT(*) FROM task;
+/*Find out how many tasks in the task table do not have a valid due date*/
+SELECT count(*) as not_valid_due_date
```
Comment (jpganz) [inline databases/week1/queries.sql:4]: this is correct :)

**Review by jpganz — APPROVED** [review]

**Pull request merged**